apps/coordinator/src/components/BCUR2/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as BCUR2Reader } from "./BCUR2Reader";
+export { default as BCUR2Scanner } from "./BCUR2Scanner";
```

Comment:
as far as I can tell the reader is the only one used outside of the module and the scanner is just imported into the reader and then re-exported.